appveyor.yml

```diff
@@ -12,7 +12,7 @@ build: off
 
 environment:
   global:
-    MAVEN_OPTS: "-Xmx1536M"
+    MAVEN_OPTS: "-Xmx2048M"
```

Comment:
Agreeded; I was hoping yours would have finished by now (and work)… meanwhile this one's been stuck waiting for AppVeyor to finish/pass. I'll pull this change as soon as we know one way or the other.